demo/index.html

```diff
@@ -37,14 +37,123 @@
   </header>
   <div class="dev-tools">
     <div id="options">
+      <button id="toggle-receipts">Toggle receipts</button>
+      <button id="toggle-buttons">Toggle button types</button>
+
+      <p id="interaction-status-area" style="min-height: 50px;margin: 10px;"></p>
     </div>
   </div>
 
   <div id="demo">
+    <style>
+      iaux-monthly-giving-circle {
+        display: block;
+        margin: 0 auto;
+        max-width: 800px;
+      }
+    </style>
     <iaux-monthly-giving-circle></iaux-monthly-giving-circle>
   </div>
 
   <script type="module" src="../dist/src/monthly-giving-circle.js"></script>
+  <script type="module">
+    import { Receipt } from '../dist/src/models/receipt.js';
+
+    let updateNotices = [];
+
+    const receiptsData = [
```

Comment:
As a practical example with types instead of classes, this can become
```
const receiptsData = [
 { amount: 9999.99, date: .... },
 { amount: 100, date: .... }
]
```
(of if you're in a TS file, `const receiptsData: aReceipt[] = [ ... ]`)